File: ADT102/Datasturcture/Queue.java

public class Queue{

		String[] i = new String[10];
		int q = 0;
		
		public void Queue(){
			for (int j =0 ; j<10; j++){
				i[j] = "0";
			}
			q = 0;
		}
		
		public void Enqueue(String a){
			i[q] = a;
			q++;
		}
		
		public void dequeue(){
			System.out.println(i[0]);
			for (int t =1; t<=q ; t++){
				i[t-1] = i[t];
			}
			q--;
		}
		
		public void isEmpty(){
			if(q!=9){
				System.out.println("true");
			}
			else{System.out.println("flase");}
		}
		
		public void front(){
			System.out.println(i[0]);
		}
		
		public void rear(){
			System.out.println(i[q-1]);
		}
		
		public void isFull() {
			if(q!=9){
				System.out.println("flase");
			}
			else{System.out.println("true");}
		}
		
		public void getSize() {
			System.out.println(q--);
		}

		  
    }

File: ADT102/AnotherThirdAssignment/Queue.py
class Queue:
    qu = []
    size = 0
    front = 0
    rear = 0

    def __init__(self, size):
        self.size = size

    def en_queue(self, data):
        self.qu.append(data)
        self.size = self.size + 1
        self.rear = self.rear + 1

    def de_queue(self):
        temp = self.qu[self.front]
        del self.qu[self.front]
        self.rear = self.rear - 1
        self.size = self.size - 1
        return temp

    def de_que_all(self):
        while not self.is_empty():
            print(self.de_queue().__str__())

    def get_front(self):
        return self.qu[self.front]

    def get_rear(self):
        return self.qu[-1]

    def is_full(self):
        return self.rear == self.get_size()

    def is_empty(self):
        return self.rear == 0

    def get_size(self):
        return self.size
